GPCS4/SceModules/SceLibkernel/sce_kernel_file.cpp

```diff
@@ -66,6 +67,26 @@ inline bool getDirName(DIR* dir, char* dirname, int len)
 
 #endif  //GPCS4_WINDOWS
 
+
+int PS4API scek__write(int fd, const void* buf, size_t size)
+{
+	LOG_SCE_TRACE("%s(fd=%d, buf=0x%p, size=%zu)", __func__, fd, buf, size);
```

Comment:
`__func__` is not needed, `LOG_SCE_TRACE` will print function name itself. Just look [what others do](https://github.com/Inori/GPCS4/blob/master/GPCS4/SceModules/SceLibkernel/sce_kernel_file.cpp#L117).